gpflow/optimizers/scipy.py

```diff
@@ -68,11 +69,13 @@ def minimize(
         )
 
     @classmethod
-    def initial_parameters(cls, variables):
+    def initial_parameters(cls, variables: Variables) -> tf.Tensor:
```

Comment:
separate question from this PR, but why are all these methods public? they don't appear to be used anywhere

Reply:
@awav would have to be the one answering that.

Reply:
@joelberkeley-pio, "They don't appear anywhere" doesn't mean that methods cannot be useful for a user. Feel free to address questions of testing and making methods protected/private in _separate_ PR.